WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsViewController.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+final class InPersonPaymentsViewController: UIHostingController<InPersonPaymentsView> {
+    init() {
+        super.init(rootView: InPersonPaymentsView())
+    }
+
+    @objc required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+struct InPersonPaymentsView: View {
+    var body: some View {
+        InPersonPaymentsUnavailableView()
+            .navigationTitle(Localization.title)
+    }
+}
+
+private enum Localization {
+    static let title = NSLocalizedString(
+        "In-Person Payments",
+        comment: "Title for the In-Person Payments settings screen"
+    )
+
+    static let unavailable = NSLocalizedString(
```

Comment:
Is this really necessary? It seems like it is not used

Reply:
Good catch, these used to be here but were moved to `InPersonPaymentsUnavailableView`